### ISSUE TITLE
[MODULE] - Deberta Sentiment Analyzer

### ISSUE THREAD

**Issue by Rashid-Ahmed** [code-kern-ai/bricks#302]
**Please describe the module you would like to add to bricks**
A sentiment Analyzer that uses a Deberta base model that is finetuned on amazon multi reviews dataset (over 1 million examples). It takes as input some string and outputs the probabilities of the text being positive, neutral and negative

**Do you already have an implementation?**

```python
import torch
from transformers import AutoModelForSequenceClassification, AutoTokenizer

def get_sentiment(sentence):
    bert_dict = {}
    vectors = tokenizer(sentence, return_tensors='pt').to(device)
    outputs = bert_model(**vectors).logits
    probs = torch.nn.functional.softmax(outputs, dim = 1)[0]
    bert_dict['neg'] = round(probs[0].item(), 3)
    bert_dict['neu'] = round(probs[1].item(), 3)
    bert_dict['pos'] = round(probs[2].item(), 3)
    return bert_dict

MODEL_NAME = 'RashidNLP/Amazon-Deberta-Base-Sentiment'
device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')

bert_model = AutoModelForSequenceClassification.from_pretrained(MODEL_NAME, num_labels = 3).to(device)
tokenizer = AutoTokenizer.from_pretrained(MODEL_NAME)

get_sentiment("This is quite a mess you have made")

```

**Additional context**
https://huggingface.co/RashidNLP/Amazon-Deberta-Base-Sentiment
https://github.com/rashid-ahmed/Sentiment_Analysis

**Comment by jhoetter** [comment]
nice one @Rashid-Ahmed !

**Comment by LeonardPuettmannKern** [comment]
Implemented with #303
